logdna/logdna.py

```diff
@@ -100,6 +101,10 @@ def send_request(self):
         self.buf.extend(self.secondary)
         self.secondary = []
         data = {'e': 'ls', 'ls': self.buf}
+        with open('VERSION.txt') as version_file:
+            pkg_version = version_file.read().strip()
+        headers = {
+            'User-Agent':'python/%s (%s %s)' % (pkg_version, platform.system(), platform.release())}
```

Comment:
No need for `platform.system()` and/or `platform.release()` - this is `codelib` - it's necessary for the agent

Reply:
I said we could have it if we wanted – happy to defer to you Samir! As long as we keep the env info in the `()` I'm down, but up to you.